package-system/pyside2-qt/Findpyside2.cmake.Linux

```diff
@@ -5,24 +5,28 @@
 #
 #
      
-# this file actually ingests the library and defines targets.
-     
 set(MY_NAME "pyside2")
 set(TARGET_WITH_NAMESPACE "3rdParty::${MY_NAME}")
 if (TARGET ${TARGET_WITH_NAMESPACE})
     return()
 endif()
-     
+
 add_library(${TARGET_WITH_NAMESPACE} INTERFACE IMPORTED GLOBAL)
-     
+
 ly_pip_install_local_package_editable(${CMAKE_CURRENT_LIST_DIR}/pyside2 pyside2)
```

Comment:
I assume this should be removed in favor of the later call with `PYSIDE_BASE_PATH`

Reply:
oh, good catch, looks like a duplicate call (see line 20)